lib/ret/media_resolver.ex

```diff
@@ -102,43 +102,34 @@ defmodule Ret.MediaResolver do
   def resolve_with_ytdl(%MediaResolverQuery{} = query, root_host, ytdl_format) do
     with ytdl_host when is_binary(ytdl_host) <- module_config(:ytdl_host) do
       case fetch_ytdl_response(query, ytdl_format) do
-        %HTTPoison.Response{status_code: 500, body: body} ->
-          # youtube-dl returns a 500 error, but includes the underlying error in its body text, search for some special cases
-          cond do
-            String.contains?(body, "is offline") ->
-              {:commit,
-               resolved(query.url, %{
-                 expected_content_type: "text/html",
-                 media_status: :offline_stream,
-                 thumbnail: RetWeb.Endpoint.static_url() <> "/stream-offline.png"
-               })}
-
-            String.contains?(body, "HTTPError 429") ->
-              {:commit,
-               resolved(query.url, %{
-                 expected_content_type: "text/html",
-                 media_status: :rate_limited,
-                 thumbnail: RetWeb.Endpoint.static_url() <> "/quota-error.png"
-               })}
-
-            true ->
-              resolve_non_video(query, root_host)
-          end
-
-        %HTTPoison.Response{status_code: 302, headers: headers} ->
-          # todo: it would be really nice to return video/* content type here!
```

Comment:
From my testing it seems to return a 500 "requested format not available" error if it can't find a video, for example if I try to look up a pdf. This is only true because we are filtering by format.
![image](https://user-images.githubusercontent.com/130735/88603473-e0aea580-d029-11ea-90d6-70d532a69556.png)